### PR TITLE
Make so that inline_links only shows up when you have the experiment enabled

### PR DESCRIPTION
**Pull request by meqativ**

### CODE REVIEW

File: plugins/urban/src/index.js [hunk 1]
```diff
@@ -1,51 +1,48 @@
...
-      execute: UrbanDef,
-      applicationId: -1,
-      inputType: 1,
-      type: 1,
-    });
-  },
-  onUnload: () => {
-    urbcmd();
-  },
+	onLoad() {
```
Comment (mugman174) [inline plugins/urban/src/index.js:6]: such goofy indentation, now the diff is messed up

**Review by mugman174 — APPROVED** [review]
I will check and merge this soon:tm:

File: plugins/urban/src/index.js (hunk 1 above)
Comment (meqativ) [reply plugins/urban/src/index.js:6]: it's just a function instead of an arrow function now

**Comment by mugman174** [comment]
what if inline links is just removed overall

**Comment by meqativ** [comment]
> what if inline links is just removed overall

what if discord add it back

**Comment by mugman174** [comment]
> > what if inline links is just removed overall
> 
> what if discord add it back

revert commit/pr maybe

**Pull request closed without merging**